app/core/api.js

```diff
@@ -458,6 +474,58 @@ api.patient.getAll = function(cb) {
   });
 };
 
+// ----- Metadata -----
+
+api.metadata = {};
+
+api.metadata.preferences = {};
+
+api.metadata.preferences.get = function(patientId, cb) {
+  tidepool.findPreferences(patientId, function(err, payload) {
+    if (err && err.status !== 404) {
```

Comment:
The `&& err.status !== 404` here and [here](https://github.com/tidepool-org/blip/pull/389/files#diff-a9e125b91e9e2c58c685083cbfa74a49R509) is a bit of an antipattern, compared to other similar functions. My reasoning is that we don't want a login or patient load to fail if they don't have any preferences/settings saved yet. Alternatives could be updating the backend prior to launch, so all accounts have empty objects for these endpoints, or setting a migration plan like [this profile update](https://github.com/tidepool-org/blip/blob/19ae3fccaff5456c871e8f058742a45175ebfebf/app/core/api.js#L188-L205), but this seemed like the cleanest option until the account eventually has preferences/settings saved. I'm not sure what our view is on making an API call (or updating a database) for what essentially is a non-operation.